docs/internal-architecture.md

```diff
@@ -0,0 +1,53 @@
+## Internal architecture
+
+There are a few resources available to understand how the Collector works:
+
+### Startup Diagram
+```mermaid
```

Comment:
I think this might just be your local setup?  If you view the "rich diff" on Github it all seems to render.  I ran into this problem locally also

Reply:
Right, I'm looking at the rendered [doc in GitHub](https://github.com/open-telemetry/opentelemetry-collector/blob/c5d84fef1917c13cbcefa44b34baa232441eade6/docs/internal-architecture.md), but some text is cut off. It may just be my browser though, if you're seeing it fine on your end 👍

Reply:
hm interesting - yeah works on my browser (lol) but this is definitely something to keep in mind if other people also have trouble viewing it